package.json

```diff
@@ -65,7 +69,10 @@
     "html-webpack-plugin": "^5.5.0",
     "mini-css-extract-plugin": "^2.6.1",
     "prettier": "^2.7.1",
+    "react-datepicker": "^4.8.0",
+    "react-icons": "^4.6.0",
```

Comment:
looks like you are no where using this dependency. If yes, remove it. Also, consider adding an associated linting rule.